sdc/tests/test_series.py

```diff
@@ -5994,6 +5998,16 @@ def test_series_islower_str(self):
             S = pd.Series(ser)
             pd.testing.assert_series_equal(cfunc(S), islower_usecase(S))
 
+    def test_series_lower_str(self):
+        series = [['leopard', 'Golden Eagle', 'SNAKE', ''],
+                  ['Hello world!', 'hello 123', 'mynameisPeter']
+                  ]
+
+        cfunc = self.jit(lower_usecase)
+        for ser in series:
+            S = pd.Series(ser)
```

Comment:
Please follow [pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

Reply:
Again, please follow [pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) when you name variables.